README.md

```diff
@@ -51,6 +53,11 @@ DATABASES = {
 }
 ```
 
+> [!TIP]
```

Comment:
This is the first time I've seen this markup. Do you know if [PyPI](https://pypi.org/project/django-mongodb-backend/) renders it correctly?